recipe/meta.yaml

```diff
@@ -3,7 +3,7 @@ package:
   version: 4.4.2
 
 build:
-  number: 0
+  number: 1
```

Comment:
We don't touch the build number on this "package". Just bump the patch number of the version (so 4.4.3).